src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -1101,6 +1085,104 @@ HRESULT STDMETHODCALLTYPE CorProfiler::JITInlining(FunctionID callerId, Function
     return S_OK;
 }
 
+
+//
+// InitializeProfiler method
+//
+void CorProfiler::InitializeProfiler(CallTargetDefinition* items, int size)
```

Comment:
nit: If this is just going to initialize the CallTarget methods, maybe this can be renamed to `InitializeCallTargetIntegration` or something similar

Reply:
I guess soon (v2.0) there will only _be_ call target, so that may not be necessary?

Reply:
I wanted to have this generic name, because I plan to do some more experimentation on passing more info between both unmanaged/managed world, and I don't want to pay the PInvoke performance penalty multiple times.